helm/prometheus-rules/templates/alerting-rules/inhibit.all.rules.yml

```diff
@@ -63,3 +63,12 @@ spec:
         team: phoenix
         topic: kiam
     {{- end }}
+    - alert: InhibitionClusterReleaseBeforeV19
+      annotations:
+        description: '{{`Cluster ({{ $labels.cluster_id }}) using Release ({{ $labels.release_version }}).`}}'
+      expr: aggregation:giantswarm:cluster_release_version{release_version!~"^(1[9]|[2-9]\\d+|[1-9]\\d{2,})\\.[0-9]{1,2}\\.[0-9]{1,2}.*$"}
+      labels:
+        release_before_v19: "true"
```

Comment:
```suggestion
        vintage_cluster_release_before_v19: "true"
```

Reply:
Is this metric only for vintage workload clusters? I'd rather we have a CAPI and non CAPI inhibition

Reply:
CAPI have no releases, we have to use something else!

Reply:
I did thought we would have some kind of concept release. You can probably use this metric:

aggregation:giantswarm:app_deployed_management_cluster_total{app="app-exporter", catalog="cluster", name="m310v-default-apps", version="0.7.3"}